reference/format/search.rst

```diff
@@ -1,2 +1,137 @@
 Search Index
 ============
+
++-----------+------------------------------------------------------------+
+| Filename  | ``search.yml``                                             |
```

Comment:
Please add example of search.yml file. It will help to understand this file structure. An example you can find by the link above.